Adding IBM Key Protect Support

Anand-Reddy7 opened the pull request:
The Storage Scale cluster file system can be encrypted using either IBM Security® Guardium® Key Lifecycle Manager (GKLM) or IBM Key Protect.

We have already integrated GKLM with the Storage Scale cluster, and now we are extending support to include the IBM Key Protect service.

rajan-mis commented:
@Anand-Reddy7 Thanks for PR. can you please add the following,
1. Overall description of PR.
2. README content for this PR
3. Sample playbooks
4. Inventory example file

Anand-Reddy7 commented:
> @Anand-Reddy7 Thanks for PR. can you please add the following,
> 
> 1. Overall description of PR.
> 2. README content for this PR
> 3. Sample playbooks
> 4. Inventory example file

Thanks @rajan-mis for pointing out the missed details.

- Added an overall description of the PR.
- Included README content for each role.
- Added sample playbooks.
- We are not creating any specific inventory for Key Protect; instead, we are using the existing Scale inventory.

If I missed anything then please let me know
